.github/workflows/linux_ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
So this will only execute Actions on a pull request to be merged into `master` and only on the "merged" head (like travis's push/pr builds but just the pr one). If that is the desired functionality you're good to go. If instead you want to use this as your main testing base for all pushes on branches and PR's you may want to change to:
```
on: [push, pull_request]
```

Reply:
Yeah, I kinda figured we don't need to run on every push. Let's leave it here for now, and we can switch if people want the tests run more frequently.